emission/core/wrapper/itinerum_boolean.py

```diff
@@ -0,0 +1,14 @@
+import emission.core.wrapper.wrapperbase as ecwb
+
+class ItinerumBoolean(ecwb.WrapperBase):
```

Comment:
To clarify, is this because we are not adding to the confirmed trip object? Gabriel and I spoke about his pipeline stage which seems to be incorporated into the confirmed trip object. How does this differ from what we are doing here?

Reply:
correct; you are not changing the trip objects since "exported or not" is not an inherent property of a trip. In Gabriel's case, the trip labels are inherently part of the trip. The exported or not is currently represented by the pipeline state object for this state.